Core/include/Acts/TrackFitting/detail/GsfActor.hpp

```diff
@@ -296,6 +296,14 @@ struct GsfActor {
       std::vector<ComponentCache> componentCache;
       convoluteComponents(state, stepper, tmpStates, componentCache);
 
+      if (componentCache.empty()) {
+        ACTS_WARNING(
+            "No components left after applying energy loss. "
+            "Is the weight cutoff too high?");
```

Comment:
```suggestion
            "Is the weight cutoff = {...} too high?");
```
could we easily print out the value of the weight cutoff here?